specs/v2-sketch.yaml

```diff
@@ -282,40 +264,29 @@ components:
               student:
                 type: boolean
                 default: false
-    Organization:
-      description: "NOTE: The definition of Organization is subject to change."
-      type: object
-      properties:
-        id:
-          type: string
-          format: uuid
     Project:
       description: "NOTE: The definition of Project is subject to change."
       type: object
+      required:
+        - id
+        - vcs_url
```

Comment:
is it required?

Reply:
This is a way of stating it will always be present, if that's not the case I'll remove it.

Reply:
I don't have a strong opinion, but feels like VCS_URL is something we _want_ to get away from generally, so if I had to pick something to put here it would probably be the slug over the URL. That said, we are nowhere close to removing it, and doing so would be a breaking change all over the place, so probably fine to have it here for now if that's simpler and more reflective of actual use. But, by "id" here we mean the slug? Are we saying that for v2 we will NEVER expose the project UUID??

Reply:
My only reservation about exposing the UUID is that it's not functionally useful, we don't have any way to use it for anything.